test/integration/users.test.js

```diff
@@ -1040,4 +1037,52 @@ describe("Users", function () {
         });
     });
   });
+
+  describe("POST /users/migrate", function () {
+    it("Should return 401 when user is unauthorize", function (done) {
+      chai
+        .request(app)
+        .post("/users/migrate")
+        .end((err, res) => {
+          if (err) {
+            return done();
+          }
+          expect(res).to.have.status(401);
+          expect(res.body.message).to.equal("Unauthenticated User");
+          return done();
+        });
```

Comment:
Aren't both these tests same?
https://github.com/Real-Dev-Squad/website-backend/blob/421c736ef0f9c4250cf28b0900c5ba26e80e4e42/test/integration/users.test.js#L1070-L1085

Reply:
I'll remove the unwanted tests.